deployment-examples/aws/serverless/extension/README.md

```diff
@@ -1,5 +1,14 @@
 ## Deploy as AWS Lambda
 
-1. For deployment to AWS Lambda, zip the extensions folder and specify `src/lambda.handler` as the entry point for the AWS Lambda function.
-2. When the extension module is run as AWS Lambda **it will NOT create** the required resources like custom types or commercetools API extension for you. 
+1. Run 'npm run zip-lambda-function'. It helps to execute following steps
+   - Rename index.lambda.js to index.js
+   - For deployment to AWS Lambda, bundle the following files and folders into a .zip file and upload it in AWS console.
+      - src
+      - node_modules
+      - resources
+      - index.js
+      - package.json
+2. Specify `index.handler` as the entry point for the AWS Lambda function.
+ > The default value in the console is index.handler which calls exports.handler in index.js
+3. When the extension module is run as AWS Lambda **it will NOT create** the required resources like custom types or commercetools API extension for you. 
```

Comment:
Now we removed the ensure thing, should we rather say they should use the script?

Reply:
Good point, this document is not the latest version of this it's not only commercetools resources I simply started with:

> In order to setup the extension module config, follow our how to run guide.

https://github.com/commercetools/commercetools-adyen-integration/tree/disable-ensure-resources/deployment-examples/aws/serverless/extension#deploy-as-aws-lambda